bin/ghe-restore

```diff
@@ -285,6 +285,16 @@ fi
 # Get GHES release version in major.minor format
 RELEASE_VERSION=$(ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config --get core.package-version' | cut -d '.' -f 1,2)
 
+
+# If the backup being restored is from an appliance with Actions disabled, restoring it onto an appliance with Actions enabled will cause
+# mismatches in the secrets needed for Actions which ultimately results in Actions not working properly.
+ACTIONS_ENABLED_IN_BACKUP=$(git config -f $GHE_RESTORE_SNAPSHOT_PATH/settings.json --list | tr . _ | grep "app_actions_enabled" | awk -F"=" '{ print $2 }')
+if [[ $ACTIONS_ENABLED_IN_BACKUP == "false" ]] && ghe-ssh "$GHE_HOSTNAME" -- 'ghe-config --true app.actions.enabled'; then
```

Comment:
Is checking "false" safe here?  What if the value is not in file?

Reply:
Good point! I have checked backups from appliances with actions enabled and disabled and this value seems to always be set. But it makes sense to account for the key not being present to be on the safe side.

Reply:
FYI: Another cool thing you can do here rather than explicitly checking for nil is use the aforementioned parameter expansion to set a default value; there are two ways:

``` bash
var=${var:-false} #explicitly set to itself or default
${var:=false} # If empty, var is assigned to default value
```

That is, if the correct behaviour is empty == false

Reply:
Updated!